Composer/packages/client/src/recoilModel/dispatchers/webchat.ts

```diff
@@ -5,14 +5,20 @@
 import { ConversationTrafficItem } from '@botframework-composer/types';
 import { useRecoilCallback, CallbackInterface } from 'recoil';
 
-import { webChatTrafficState, webChatInspectionDataState, isWebChatPanelVisibleState } from '../atoms';
+import {
+  webChatTrafficState,
+  webChatInspectionDataState,
+  isWebChatPanelVisibleState,
+  watchedVariablesState,
+} from '../atoms';
 import { WebChatInspectionData } from '../types';
 
 export const webChatLogDispatcher = () => {
   const clearWebChatLogs = useRecoilCallback((callbackHelpers: CallbackInterface) => (projectId: string) => {
     const { set } = callbackHelpers;
     set(webChatTrafficState(projectId), []);
     set(webChatInspectionDataState(projectId), undefined); // clear the inspection panel
+    set(watchedVariablesState(projectId), []); // TODO: might not want to do this depending on how annoying it is for the user -- do you want to wipe variables when you restart convo?
```

Comment:
@tonyanziano We definitely dont want to wipe out the variable list on bot restart like you said following the VSCode model